### PR TITLE
added --expand-head-model arg + argparser

### PR DESCRIPTION
**Pull request by jenholmberg**

### CODE REVIEW

File: make_headcase.py [hunk 1]
```diff
@@ -221,10 +221,10 @@ def align_scan(infile, outfile):
 def gen_case(
     scanfile,
     outfile,
+    expand_head_model,
```
Comment (mvdoc) [inline make_headcase.py:224]: Please revert it back to being a keyword argument with a 0.1 default

File: make_headcase.py [hunk 2]
```diff
@@ -267,7 +267,7 @@ def gen_case(
     if workdir is None:
         workdir = mkdtemp()
         cleanup = True
-
+    print(expand_head_model)
```
Comment (mvdoc) [inline make_headcase.py:270]: remove this line

File: make_headcase.py [hunk 3]
```diff
@@ -364,6 +367,7 @@ def pipeline(
     gen_case(
         aligned,
         outfile,
+        expand_head_model,
```
Comment (mvdoc) [inline make_headcase.py:370]: make it a keyword argument with default 0.1 and put it at the end

File: make_headcase.py [hunk 4]
```diff
@@ -451,12 +455,14 @@ def pipeline(
...
     customizations = args.customizations_file
     generate_headcase_only = args.generate_headcase_only
+    expand_head_model = args.expand_head_model
 
     if generate_headcase_only:
         print("Making head case")
         gen_case(
             infile,
             outfile,
+            expand_head_model,
```
Comment (mvdoc) [inline make_headcase.py:465]: once it's a keyword argument in `gen_case`, change this to `expand_head_model=expand_head_model`

**Review by mvdoc — CHANGES_REQUESTED** [review]

File: make_headcase.py (hunk 1 above)
Comment (jenholmberg) [reply make_headcase.py]: done

File: make_headcase.py (hunk 2 above)
Comment (jenholmberg) [reply make_headcase.py]: woops, done

File: make_headcase.py (hunk 4 above)
Comment (jenholmberg) [reply make_headcase.py]: done

**Pull request merged**